showcase/image_classification/lenet_cifar10_mixup.py

```diff
@@ -24,28 +24,20 @@
 from fastestimator.pipeline.pipeline import Pipeline
 from fastestimator.pipeline.preprocess import Minmax
 
-
```

Comment:
your IDE isn't following pep8 formatting standards - there should be 2 spaces here